web/src/components/tasks/business-formation/BusinessFormation.tsx

```diff
@@ -100,7 +100,7 @@ export const BusinessFormation = (props: Props): ReactElement => {
         business.formationData.formationFormData.businessStartDate,
         legalStructureId
       ),
-      addressMunicipality: business.profileData.municipality,
+      addressMunicipality: business.formationData.formationFormData.addressMunicipality,
```

Comment:
[dust] I'm a bit confused about this change

Was there a refactor changing how where the address municipality was stored? Was in previously in profile data and is now in formationFormData?

I don't see changes to userData and the types, so was this just an area that was missed in a prior refactor?

Reply:
Yes, Aaron, we found that odd as well (that all of the other values pointed to `formationData.formationFormData`). `addressMunicipality` was the only one set to `profileData`.

Reply:
I'm marking this unresolved. I'm not quite sure what the difference is between the profile municipality and he one on formationFormData. They're both used throughout the application. We should validate the purpose of this before assuming it's safe to swap.

Reply:
`profileData.municipality` is for the profile, and `formationFormData. municipality` is for business formation. If we leave `addressMunicipality` set to `profileData.municipality`, the `city` field in business formation will remain empty, even after users enter the city in their profile. 

In other words, when users enter their address in the profile, all the address data persists from the profile _EXCEPT_ the city. Updating the value to `formationData.formationFormData.addressMunicipality` ensures the city persists in business formation.

Reply:
Here is the updated AC describing the requested change. I think we need to think about the implications of this change and confirm with product that this is the correct choice forward.

Anju - formation has a municipality field and a city field.


A/C
Given I make changes to my business address in the Business Profile
When reviewing my business address in the formation task
Then the Business Address fields should reflect the same change (pre-populated) and the field validations continue to function (implemented previously)

Reply:
`addressCity` is for foreign formation and `addressMunicipality` is for domestic formation. We have an outstanding refactor ticket to rename the the fields to avoid confusion: [Refactor: rename 'addressCity' and 'addressMunicipality' in formation](https://www.pivotaltracker.com/n/projects/2573455/stories/184036514)